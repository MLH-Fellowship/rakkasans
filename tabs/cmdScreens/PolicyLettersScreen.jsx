import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import PDFReader from 'rn-pdf-reader-js';
import { Asset } from 'expo-asset';

import Colors from '../../constants/Colors';
import RectButton from '../../components/RectButton';
import Dimensions from '../../constants/Dimensions';

export default function PolicyLettersScreen() {
  // let absolute_path = Asset.fromModule(
  //   require("../../assets/pdfs/POLICY_1.pdf")
  // ).uri;
  return (
    <View style={styles.container}>
      <RectButton text="Policy Letter 1" />
      <RectButton text="Policy Letter 2" />
      <RectButton text="Policy Letter 3" />
      <RectButton text="Policy Letter 4" />
      <RectButton text="Policy Letter 5" />
      <RectButton text="Policy Letter 6" />
      <RectButton text="Policy Letter 7" />
    </View>
    // <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
    //   <PDFReader
    //     style={styles.pdf}
    //     source={{
    //       uri: "http://gahp.net/wp-content/uploads/2017/09/sample.pdf",
    //     }}
    //   />
    // </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    paddingHorizontal: 20,
  },
  // pdf: {
  //   width: "100%",
  //   height: "100%",
  // },
});
