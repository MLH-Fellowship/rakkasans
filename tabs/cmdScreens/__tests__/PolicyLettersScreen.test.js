import React from 'react';
import renderer from 'react-test-renderer';

import PolicyLettersScreen from '../PolicyLettersScreen';

jest.useFakeTimers();

it('renders correctly', () => {
  const tree = renderer.create(<PolicyLettersScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
