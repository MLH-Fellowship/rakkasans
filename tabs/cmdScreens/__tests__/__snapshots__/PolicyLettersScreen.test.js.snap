// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "paddingHorizontal": 20,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 1
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 2
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 3
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 4
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 5
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 6
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#020a41",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 10,
        "opacity": 1,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#ffffff",
            "fontSize": 18,
            "marginHorizontal": 30,
            "marginTop": 7,
          },
          Object {
            "fontFamily": "andale-mono",
          },
        ]
      }
    >
      Policy Letter 7
    </Text>
  </View>
</View>
`;
